website/server/models/group.js

```diff
@@ -577,6 +578,18 @@ schema.statics.cleanGroupQuest = function cleanGroupQuest () {
   };
 };
 
+async function _updateUserWithRetries (userId, updates, numTry = 1) {
+  return User.update({_id: userId}, updates).exec((err, raw) => {
```

Comment:
Is there a reason to use the callback syntax instead of the promise syntax? If I'm reading this correctly, returning these values isn't actually doing anything.